universal-application-tool-0.0.1/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -76,6 +76,29 @@ public boolean isOptional() {
     return programQuestionDefinition.optional();
   }
 
+  /**
+   * Return true if this question is answered, or left unanswered in the program specified.
+   * Questions can only be left unanswered if they are optional.
+   *
+   * <p>For every applicant question, there are three possible states:
+   *
+   * <ol>
+   *   <li>unvisited - the applicant has not seen this question yet
+   *   <li>unanswered - the applicant has visited the question but chose to leave it unanswered.
+   *       Only optional questions can be left unanswered
+   *   <li>answered - the applicant has visited and provided an answer to the question
+   * </ol>
```

Comment:
Per my previous question about having some comments capturing broader design, it'd be helpful to add some description here on how static questions are handled.

Reply:
Added static questions here